package/boot/uboot-envtools/files/qualcommax_ipq60xx

```diff
@@ -26,6 +26,9 @@ netgear,wax214|\
 tplink,eap610-outdoor)
 	ubootenv_add_mtd "0:appsblenv" "0x0" "0x40000" "0x20000"
 	;;
+tplink,eap620hd-v2)
+	ubootenv_add_mtd "0:appsblenv" "0x0" "0x40000" "0x20000"
```

Comment:
This is the same as above (`netgear,wax214` and `tplink,eap610-outdoor`), please merge them.